go/oasis-test-runner/scenario/e2e/runtime/runtime_client.go

```diff
@@ -0,0 +1,127 @@
+package runtime
```

Comment:
Should this interface and all the clients move into its own package?

Reply:
No, since the concrete implementations require reaching into the runtimeImpl (and none of this code should be reused outside of testing ever).